Tests/SonarQube.TeamBuild.PreProcessor.Tests/ArgumentProcessorTests.cs

```diff
@@ -32,38 +32,31 @@ public void PreArgProc_MissingArguments()
             // 1. Null logger
             AssertException.Expects<ArgumentNullException>(() => ArgumentProcessor.TryProcessArgs(null, null));
 
-            using (EnvironmentVariableScope scope = new EnvironmentVariableScope())
-            {
-                CreateRunnerFilesInScope(scope);
-
```

Comment:
Simplified the tests: they no longer need to be able to locate a sonar-runner.properties file to pass.